### PR TITLE
Slim snippet

### PR DESCRIPTION
**Pull request by hyanwong**
In the blurb, I cut the bit about Eidos for brevity, but it could be reintroduced if we'd like more text.

### CODE REVIEW

File: _software/SLiM.md [hunk 1]
```diff
@@ -7,6 +7,21 @@ docs_url: https://messerlab.org/slim/
...
+  1 { sim.addSubpop("p1", 500); }
+  2000 late() { sim.treeSeqOutput("out.trees"); }
 ---
-Here is some content that is on the full page about SLiM
\ No newline at end of file
+SLiM is an evolutionary simulation framework that combines a powerful engine
+for population genetic simulations with the capability of modeling arbitrarily
+complex evolutionary scenarios. The underlying individual-based simulation
+engine is highly optimized to enable modeling of entire chromosomes in
+large populations. There is also a graphical user interface on macOS and Linux
```
Comment (petrelharp) [inline _software/SLiM.md:25]:
```suggestion
large populations. There is also a graphical user interface
```
People run it on Windows also...

File: _software/SLiM.md [hunk 2]
```diff
@@ -7,6 +7,21 @@ docs_url: https://messerlab.org/slim/
...
+      initializeRecombinationRate(1e-8);
+  }
+  1 { sim.addSubpop("p1", 500); }
+  2000 late() { sim.treeSeqOutput("out.trees"); }
 ---
-Here is some content that is on the full page about SLiM
\ No newline at end of file
+SLiM is an evolutionary simulation framework that combines a powerful engine
+for population genetic simulations with the capability of modeling arbitrarily
+complex evolutionary scenarios. The underlying individual-based simulation
```
Comment (petrelharp) [inline _software/SLiM.md:23]:
```suggestion
SLiM is a forwards-time population genetics simulator capable of simulating arbitrarily
complex evolutionary scenarios. The underlying individual-based simulation
```

**Comment by petrelharp** [comment]
Looks great, thanks!

File: _software/SLiM.md (hunk 1 above)
Comment (hyanwong) [reply _software/SLiM.md:25]: True - this is simply copied straight off the main SLiM website - you might want to point out to Ben that he can change that there too?

**Review by benjeffery — APPROVED** [review]

**Pull request merged**